src/bindings/overlay.js

```diff
@@ -95,13 +130,80 @@ function overlay(l10n, sourceElement, translationElement) {
   // cleared if a new language doesn't use them; https://bugzil.la/922577
   if (translationElement.attributes) {
     for (k = 0, attr; (attr = translationElement.attributes[k]); k++) {
-      if (l10n.isAttrAllowed(attr, sourceElement)) {
+      if (isAttrAllowed(attr, sourceElement)) {
         sourceElement.setAttribute(attr.name, attr.value);
       }
     }
   }
 }
 
+/**
+ * Check if element is allowed in the translation.
+ *
+ * This method is used by the sanitizer when the translation markup contains
+ * an element which is not present in the source code.
+ *
+ * @param   {Element} element
+ * @returns {boolean}
+ * @private
+ */
+function isElementAllowed(element) {
+  const allowed = ALLOWED_ELEMENTS[element.namespaceURI];
+  if (!allowed) {
+    return false;
+  }
+
+  return allowed.indexOf(element.tagName.toLowerCase()) !== -1;
```

Comment:
I don't think we can yet, sadly.  It's not well supported and IIRC Babel doesn't transpile it unless you use a special plugin and then it only work for array literals anyways (`['a', 'b'].includes('a')`).